ebpf-io/ebpf.io-website
# feat: adding google calendar integration to events page

**This PR adding Events page**
TODO: 

- [x] Develop main structure and UI;
- [x] Add filtering;
- [x] Add gatsby-source-google-calendar plugin;
- [ ] Add link to CardWithCta
- [ ] Integrate eCHO events.
- [ ] Change Google Calendar data in Netlify

**Steps to test**

- Open the [page](https://deploy-preview-379--ebpfio.netlify.app/events)
- Confirm that everything looks and works as expected
